docs/advanced/addons/vmimport.md

```diff
@@ -212,3 +212,7 @@ OpenStack allows users to have multiple instances with the same name. In such a
 
 #### Known issues
 * **Source virtual machine name is not RFC1123 compliant**: When creating a virtual machine object, the vm-import-controller add-on uses the name of the source virtual machine, which may not meet the Kubernetes object [naming criteria](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names). You may need to rename the source virtual machine to allow successful completion of the import.
+* **v1.6.0-rc3 vm-import-controller gracefulShutdownTimeoutSeconds, once it is exceeded a forceful shutdown is not performed and VM never migrates from VMware**: When creating a VirtualMachineImport and using a VMware source, if VMware Tools are not present on the VM a graceful shutdown can not be completed, the VM will not migrate. You will be required to one of the following as a work-around:
```

Comment:
```suggestion
* **v1.6.0 vm-import-controller gracefulShutdownTimeoutSeconds, once it is exceeded a forceful shutdown is not performed and VM never migrates from VMware**: When creating a VirtualMachineImport and using a VMware source, if VMware Tools are not present on the VM a graceful shutdown can not be completed, the VM will not migrate. You will be required to one of the following as a work-around:
```

Reply:
great catch :smile: - I've updated and forced pushed up the changes :+1: